### PR TITLE
Added test for limitseq() (Issue#11672)

### LINKED ISSUE #11672: limit(Rational(-1,2)**k, k, oo) fails
Python 3.5.2, Sympy 1.0.

The following code

```
k = symbols('k', integer = True, positive= True)
limit(Rational(-1,2)**k, k, sp.oo)
```

returns `NotImplementedError`, instead of `0`.

### PR DESCRIPTION
**Pull request by goddus**
Fixes #11672

NO ENTRY

### CODE REVIEW

File: sympy/series/tests/test_limitseq.py [hunk 1]
```diff
@@ -125,6 +125,10 @@ def test_issue_10382():
     assert limit_seq(fibonacci(n+1)/fibonacci(n), n) == S.GoldenRatio
 
 
+def test_issue_11672():
+    assert limit_seq(Rational(-1,2)**n, n) == 0
```
Comment (sachin-4099) [inline sympy/series/tests/test_limitseq.py:129]: There could be a space after the comma inside `Rational()` expression.

**Comment by sachin-4099** [comment]
The release notes should have `NO ENTRY` I guess.

File: sympy/series/tests/test_limitseq.py [hunk 2]
```diff
@@ -125,8 +125,8 @@ def test_issue_10382():
     assert limit_seq(fibonacci(n+1)/fibonacci(n), n) == S.GoldenRatio
 
 
-def test_issue_16735():
-    assert limit_seq(5**n/factorial(n), n) == 0
```
Comment (sachin-4099) [inline sympy/series/tests/test_limitseq.py:129]: This testcase should not be deleted.

**Comment by Smit-create** [comment]
@goddus Add `Fixes #issue_number` in the release notes so that the issue gets closed upon merging this automatically

**Comment by sachin-4099** [comment]
> @goddus Add `Fixes #issue_number` in the release notes so that the issue gets closed upon merging this automatically

By this he meant, adding it as a part of the comment but not in the release notes literally. The release notes should has `NO ENTRY` only,

**Comment by sachin-4099** [comment]
Your PR is adding some unnecessary files. I will have to request for a new PR, as this is getting messy and so many commits are not required for a single testcase. Sorry for the inconvenience. I think with a new PR, you will not require so many commits, one should be enough.

**Comment by goddus** [comment]
Of course

**Comment by sachin-4099** [comment]
Closing the PR in favour of: https://github.com/sympy/sympy/pull/19740#issuecomment-657102315

**Pull request closed without merging**